### ISSUE TITLE
Help needed: Error installing supervised on Raspberry Pi 4 on Bullseye

### ISSUE THREAD

**Issue by uswong** [home-assistant/supervised-installer#378]
### OS Version

Raspbian GNU/Linux 11 (bullseye)

### System Information

Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux

### What happened?

I encountered ERROR while trying to install Home Assistant Supervised on my Raspberry pi 4 on Bullseye.  RaspBerry Pi 4 OS was freshly installed from Raspberry pi installer image.  The installation message has the following error:

`[error] Raspbian GNU/Linux 11 (Bullseye) is not supported`

Similar message when I tried it on a freshly installed Bookworm (Raspbian GNU/Linux 12):

`[error] Raspbian GNU/Linux 12 (Bookworm) is not supported`

I have followed exactly the installation instructions outlined in this page.

Please help!

### Machine Type

raspberrypi4

### Installer output

```text
pi@raspberrypi:~ $ sudo apt install ./homeassistant-supervised.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'homeassistant-supervised' instead of './homeassistant-supervised.deb'
The following NEW packages will be installed:
  homeassistant-supervised
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/7,432 B of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 /home/pi/homeassistant-supervised.deb homeassistant-supervised all 1.6.0 [7,432 B]
Preconfiguring packages ...                   
(Reading database ... 107224 files and directories currently installed.)
Preparing to unpack .../homeassistant-supervised.deb ...
[warn] 
[warn] If you want more control over your own system, run
[warn] Home Assistant as a VM or run Home Assistant Core
[warn] via a Docker container.
[warn] 
[error] Raspbian GNU/Linux 11 (bullseye) is not supported!
dpkg: error processing archive /home/pi/homeassistant-supervised.deb (--unpack):
 new homeassistant-supervised package pre-installation script subprocess returned error exit status 1
[info] Undo divert on abort-install
No diversion 'diversion of /etc/NetworkManager/NetworkManager.conf to /etc/NetworkManager/NetworkManager.conf.real by homeassistant-supervised', none removed.
No diversion 'diversion of /etc/NetworkManager/system-connections/default to /etc/NetworkManager/system-connections/default.real by homeassistant-supervised', none removed.
No diversion 'diversion of /etc/docker/daemon.json to /etc/docker/daemon.json.real by homeassistant-supervised', none removed.
No diversion 'diversion of /etc/network/interfaces to /etc/network/interfaces.real by homeassistant-supervised', none removed.
[info] Resetting debconf selections
[info] Removal complete, due to the complexity of this installation method,
[info] you will need to manually remove the containers created by the supervisor
Errors were encountered while processing:
 /home/pi/homeassistant-supervised.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

### Relevant log output

```shell
Sep 21 09:31:21 raspberrypi sudo[1753]: pam_unix(sudo:session): session closed for user root
Sep 21 09:31:53 raspberrypi sudo[1900]:       pi : TTY=pts/0 ; PWD=/home/pi ; USER=root ; COMMAND=/usr/bin/apt install ./homeassistant-supervised.deb
Sep 21 09:31:53 raspberrypi sudo[1900]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1000)
Sep 21 09:31:59 raspberrypi sudo[1900]: pam_unix(sudo:session): session closed for user root
```

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by ikifar2012** [comment]
Raspberry Pi OS is not supported... Try one of the images listed here: https://raspi.debian.net/tested-images/

**Comment by uswong** [comment]
Thanks @ikifar2012 . 

I installed HomeAssistant Supervised on the very Raspberry Pi 4 two years ago and was running like a charm until I accidentally did something that I am not supposed to and have to reinstall the Raspberry Pi OS.  I tried to install it again on freshly installed OS exactly the same way as before but failed this time.  I wonder when they decided not to support Raspberry Pi OS anymore!

In any case, I decided to try other methods and installed HomeAssistant Container instead (https://www.home-assistant.io/installation/raspberrypi-other) and was as easy and work as well.
